### DISCUSSION TITLE
My organization in GitHub changed to Free Plan

### DISCUSSION THREAD

**Discussion by miltondp** [community/community#55927] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm an Assistant Professor, and recently I was approved as a teacher in GitHub Education. We use GitHub a lot to develop open source/open science projects. When I was approved as a teacher, I was able to upgrade [my lab's organization](https://github.com/pivlab) to the GitHub Team plan. However, a few weeks ago, it switched back to a Free Plan without any reason (I never received an email, etc).

If I try to use the GitHub Global Campus dashboard to convert again to the Team Plan, I don't see my lab's organization listed, so I cannot change it. How can I get the Team Plan back? There are many features that we need to use for computational research development.

Thank you.

Other related issues I found:
* https://github.com/orgs/community/discussions/53609
* https://github.com/orgs/community/discussions/50516

**Comment by samus-aran** [comment]
Hi @miltondp 

Are you still experiencing this problem? Please let us know and the community team be happy to assist 👍

**Reply by miltondp** [reply]
Hi. I had to recreate my organization to fix the problem, which is of course not a solution.

Thank you.
